code/modules/spells/spell_types/wizard.dm

```diff
@@ -354,4 +354,36 @@
 		var/mob/living/carbon/C = user
 		playsound(C.loc, 'sound/voice/hiss5.ogg', 80, 1, 1)
 		C.spin(6,1)
+		playsound(C.loc, 'sound/effects/hit_kick.ogg', 100, 1, 1)
+	var/list/thrownatoms = list()
+	var/atom/throwtarget
+	var/distfromcaster
+	for(var/turf/T in targets) //Done this way so things don't get thrown all around hilariously.
+		for(var/atom/movable/AM in T)
+			thrownatoms += AM
+
+	for(var/atom/movable/AM in thrownatoms)
+		if(AM == user || AM.anchored) continue
+
+		var/obj/effect/overlay/targeteffect	= new /obj/effect/overlay{icon='icons/effects/effects.dmi'; icon_state="shieldsparkles"; mouse_opacity=0; density = 0}()

```

Comment:
And in English?

This is literally just copy pasted from the normal repulse spell because I couldn't figure out how to change the stun durations without just repeating the same code used by repulse.